pkg/occlient/occlient_test.go

```diff
@@ -1280,7 +1280,7 @@ func TestGetBuildConfig(t *testing.T) {
 					t.Errorf("wrong GetBuildConfig got: %v, expected: %v", build.Name, tt.buildName)
 				}
 			} else if err == nil && tt.wantErr {
-				t.Errorf("test failed, expected: %s, got %s", "error", "no error")
+				t.Error("test failed, expected: error, got no error")
```

Comment:
When you see just this error it is not clear what happened.
"error was expected, but no error returned" or something like this